### PR TITLE
Add tool to fetch PR information based on assigned labels

### PR DESCRIPTION
**Pull request by benedikt-voelkel**
* applicable to different repos

* distinguishes between merged and other (simply closed or still open) PRs

* dumps the output into a simple text file for further proessing

### CODE REVIEW

**Comment by benedikt-voelkel** [comment]
For now it can fetch PRs based on labels. More functionality can be added incrementally.
E.g.
```bash
python3 UTILS/o2dpg_report.py --owner AliceO2Group --repo O2DPG --prod-label 2022pp_apass4 --date 19.02.2024 --per-page 100 --pages 3 --output o2dpg_report_prs.txt
```
gives
# FROM OLDEST TO RECENT
| Date of next tag | Requestor | Package | PR | Data or MC | Comment | JIRA (if it exists) | Accepted | In production | Validated by requestor |
| ---------------- | ------------ | ------- | --------------------------------------------------------:|:--------------------------------------------- | ------------------- | ---------------- | ------------- |-------------| ------------------|
| 19.02.2024 | benedikt-voelkel | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1358) | MC |  Align SOR and first orbit with CTPScalers |  | | |  |
| 19.02.2024 | benedikt-voelkel | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1416) | MC |  [SimWF] Always set a process for Pythia8 |  | | |  |
| 19.02.2024 | catalinristea | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1340) | MC |  new anchoredMC.sh for nowadays productions |  | | |  |
| 19.02.2024 | njacazio | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1419) | MC |  Update hadronic cross section in injected MC  |  | | |  |
| 19.02.2024 | catalinristea | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1420) | MC |  Update anchorMC.sh - removing the use of ideal MFT/ITS alignments |  | | |  |
| 19.02.2024 | sawenzel | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1432) | MC |  Possibility to take external config for Pythia8 |  | | |  |
| 19.02.2024 | benedikt-voelkel | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1437) | MC |  [SimWF] Comments for anchored WF script |  | | |  |
| 19.02.2024 | sawenzel | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1452) | MC |  Anchoring: Ability to get detector list from GRPECS |  | | |  |
| 19.02.2024 | benedikt-voelkel | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1441) | MC |   [SimWF] Update reco and sources |  | | |  |
| 19.02.2024 | catalinristea | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1462) | MC |  Update anchorMC.sh - added proc arg |  | | |  |
| 19.02.2024 | benedikt-voelkel | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1466) | MC |  Revert "Update anchorMC.sh - added proc arg" |  | | |  |
| 19.02.2024 | sawenzel | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1443) | MC |  Fix type error in pipeline runner |  | | |  |
| 19.02.2024 | sawenzel | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1471) | MC |  pipeline_runner: Fix script creation |  | | |  |
| 19.02.2024 | benedikt-voelkel | O2DPG | [PR](https://github.com/AliceO2Group/O2DPG/pull/1463) | MC |  [AnchorMC] Allow for any additional option for sim WF |  | | |  |

File: UTILS/o2dpg_report.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python3
```
Comment (sawenzel) [inline UTILS/o2dpg_report.py:1]: fine ... but wouldn't call this tool `o2dpg_report`. Preferably, name should tell about PR / github / ... functionality.

Comment (benedikt-voelkel) [reply UTILS/o2dpg_report.py]: Makes sense, renamed it

**Comment by chiarazampolli** [comment]
Nice!!

**Pull request merged**